migration/migrator/migrations/course/20240626171518_remove_poll_status.py

```diff
@@ -0,0 +1,33 @@
+"""Migration for a given Submitty course database."""
+
+
+def up(config, database, semester, course):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE ONLY polls DROP COLUMN IF EXISTS status")
+
+
+def down(config, database, semester, course):
+    """
+    Run down migration (rollback).
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    :param database: Object for interacting with given database for environment
+    :type database: migrator.db.Database
+    :param semester: Semester of the course being migrated
+    :type semester: str
+    :param course: Code of course being migrated
+    :type course: str
+    """
+    database.execute("ALTER TABLE ONLY polls ADD COLUMN IF NOT EXISTS status TEXT NOT NULL")
```

Comment:
The column was nullable before, so shouldn't you drop the not null constraint here?

Reply:
changed in [caf8eac](https://github.com/Submitty/Submitty/pull/10626/commits/caf8eac1dc7294871046c851350f15781819e2c1)